tools/deb/istio-start.sh

```diff
@@ -39,8 +39,11 @@ ISTIO_CFG=${ISTIO_CFG:-/var/lib/istio}
 NS=${ISTIO_NAMESPACE:-default}
 SVC=${ISTIO_SERVICE:-rawvm}
 ISTIO_SYSTEM_NAMESPACE=${ISTIO_SYSTEM_NAMESPACE:-istio-system}
-ISTIO_PILOT_PORT=${ISTIO_PILOT_PORT:-15003}
-CONTROL_PLANE_AUTH_POLICY=${CONTROL_PLANE_AUTH_POLICY:-MUTUAL_TLS}
+
+# The default matches the default istio.yaml - use sidecar.env to override this if you
```

Comment:
I'm torn about that one, I think auth (smart enough to not break things) should be the default - specially for mesh expansion where we want to join securely

Reply:
Agree - however it is not the default today in istio.yaml.

See my comment - for 0.6 we should make 15003 mTLS by default. We still need to migrate,
and support cases where people don't do it all at once.